tests/data/inria/data.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+
+import numpy as np
+import rasterio as rio
+from rasterio.crs import CRS
+from rasterio.transform import Affine
+
+from torchvision.datasets.utils import calculate_md5
+
+
+def write_data(
+    path: str, img: np.array, driver: str, crs: CRS, transform: Affine
```

Comment:
I think it should `np.ndarray`. `np.array` is a function and `np.ndarray` is the actual class.